content/en/docs/languages/go/getting-started.md

```diff
@@ -409,30 +401,6 @@ func init() {
 		panic(err)
 	}
 }
-
-func rolldice(w http.ResponseWriter, r *http.Request) {
```

Comment:
This has been moved to https://github.com/open-telemetry/opentelemetry-go/blob/main/example/dice/rolldice.go so we need to update the documentation accordingly